### DISCUSSION TITLE
Landtamic Pro | Local Installation

### DISCUSSION THREAD

**Discussion by horitani** [lucky-media/landtamic-pro#2] [Q&A] [answered]
Hello,

If we install Landtamic Pro locally for development, we have to enter the license key - this validates the license and I can then no longer install the package on the live server or a stage. What would be the correct procedure here?

Thank you!
Jesser

**Comment by lokmanm** [comment]
Hey @horitani,

The starter kits are not add-ons. Once you install it on your local dev machine you can easily deploy anywhere and it won't ask for a license. 

Here is a quick article on Addons vs Starter Kits:
https://statamic.dev/extending/addons#addons-vs-starter-kits

Hope that helps!

**Reply by horitani** [reply]
Hello,

thank you for your response. Unfortunately the first approach to install the starter kit failed due to local environment issues, but the registration triggered anyway. Could you please reset the registration once, so i can repeat the installation process ?

Thank you!

**Reply by lokmanm** [reply, marked as answer]
Hey @horitani,

As i mentioned over email, we don't handle License issues. You will need to email support@statamic.com and they will issue a new license.
